README.md

```diff
@@ -7,916 +7,53 @@
 
-  - [General model](#general-model)
-  - [Layer configuration examples](#layer-configuration-examples)
-  - [Using Pebble](#using-pebble)
-  - [Container usage](#container-usage)
-  - [Layer specification](#layer-specification)
-  - [API and clients](#api-and-clients)
-  - [Hacking / Development](#hacking--development)
-  - [Contributing](#contributing)
+Pebble's key features:
 
-## General model
+- [Layer](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/layers/)-based configuration
+- Service [dependencies](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/service-dependencies/)
+- Service [logs](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/logs/) and [log forwarding](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/log-forwarding/)
+- [Health checks](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/health-checks/)
+- [Notices](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/notices/) (aggregated events)
+- Identities
+- Can be used in machines and [containers](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/pebble-in-containers/)
+- [CLI commands](https://canonical-pebble.readthedocs-hosted.com/en/latest/reference/cli-commands/cli-commands/)
+- [HTTP API](https://canonical-pebble.readthedocs-hosted.com/en/latest/explanation/api-and-clients/) with a [Go client](https://pkg.go.dev/github.com/canonical/pebble/client) and a [Python client](https://github.com/canonical/operator/blob/main/ops/pebble.py)
 
-Pebble is organized as a single binary that works as a daemon and also as a
-client to itself. When the daemon runs it loads its own configuration from the
-`$PEBBLE` directory, as defined in the environment, and also records in
-that same directory its state and unix sockets for communication. If that variable
-is not defined, Pebble will attempt to look for its configuration from a default
-system-level setup at `/var/lib/pebble/default`. Using that directory is encouraged
-for whole-system setup such as when using Pebble to control services in a container.
+## Quick start
 
-The `$PEBBLE` directory must contain a `layers/` subdirectory that holds a stack of
-configuration files with names similar to `001-base-layer.yaml`, where the digits define
-the order of the layer and the following label uniquely identifies it. Each
-layer in the stack sits above the former one, and has the chance to improve or
-redefine the service configuration as desired.
+Prerequisites:
 
-## Layer configuration examples
+- A Linux machine.
```

Comment:
This kinda reads funny to have only a single item rerequisite bulleted list. Perhaps remove "Prerequisites:" and "- A linux machine." and just introduce it with something like this:

```
At any Linux shell:
```